src/chainparams.cpp

```diff
@@ -155,8 +155,8 @@ class CMainParams : public CChainParams {
         nPruneAfterHeight = 100000;
         nAuxpowChainId = 20;
         nChainStallDuration = 60*60;
-        m_assumed_blockchain_size = 350;
-        m_assumed_chain_state_size = 6;
+        m_assumed_blockchain_size = 9;
```

Comment:
I think this is OK as a start. Future wallets, especially with the SPV drive may result in smaller numbers